EU4toV2/Source/Mappers/CultureMapper/CultureMapper.h

```diff
@@ -25,10 +29,24 @@ namespace mappers
 			int eu4Province,
 			const std::string& eu4ownerTag) const;
 
+		[[nodiscard]] std::optional<std::string> cultureRegionalMatch(
+			const EU4::Regions& eu4Regions,
+			const std::string& eu4culture,
+			const std::string& eu4religion,
+			int eu4Province,
+			const std::string& eu4ownerTag) const;
+
+		[[nodiscard]] std::optional<std::string> cultureNonRegionalNonReligiousMatch(
+			const EU4::Regions& eu4Regions,
+			const std::string& eu4culture,
+			const std::string& eu4religion,
+			int eu4Province,
+			const std::string& eu4ownerTag) const;
+
 	private:
 		void registerKeys();
 		
-		std::vector<CultureMappingRule> cultureMapRules;
+		std::list<CultureMappingRule> cultureMapRules;
```

Comment:
Due to caching, it seems as if vector is actually faster: https://baptiste-wicht.com/posts/2012/12/cpp-benchmark-vector-list-deque.html

Probably best to keep it (unless you saw performance differences favoring list, in which case disregard).

Reply:
yeah, was about to shuffle rules internally according to distinguishers but it turned out unnecessary. fixed